ui/packages/ui/tslint.json

```diff
@@ -7,6 +7,7 @@
   },
   "rules": {
     "jsx-no-lambda": false,
+    "ordered-imports": false,
```

Comment:
I think we could leave the ordered-import true, as vscode(lint plugins) itself provides a one-click fix for it.